### PR TITLE
Computing of crops corrected

### PR DESCRIPTION
**Pull request by weaselix**

### CODE REVIEW

File: apps/rendering/resources/scripts/img_metrics_calculator.py [hunk 1]
```diff
@@ -31,76 +39,41 @@ def calculate_metrics(reference_img_path,
...
     :return:
     """
-
-    # pylint: disable=too-many-locals
-
+    metrics_path = metrics_output_filename
     cropped_img, scene_crops, _rendered_scene = \
         _load_and_prepare_images_for_comparison(reference_img_path,
                                                 result_img_path,
                                                 xres,
```
Comment (magdasta) [inline apps/rendering/resources/scripts/img_metrics_calculator.py]: The name of this variable is very misleading, it has nothing to do with resolution. Please find better name and change it in all functions in this file.

Comment (magdasta) [reply apps/rendering/resources/scripts/img_metrics_calculator.py]: and, of course, analogically for `yres`

File: apps/rendering/resources/scripts/img_metrics_calculator.py [hunk 2]
```diff
@@ -156,14 +129,10 @@ def convert_to_png_if_needed(img_path):
...
-    for x_offset in offsets:
-        for y_offset in offsets:
-            crop = rendered_scene.crop(
-                (x + x_offset, y + y_offset, x + width - x_offset,
-                 y + height - y_offset))
-            crops.append(crop)
+    offsets = itertools.product([0, -1, 1], repeat=2)
+    crops = {(x_offset, y_offset): rendered_scene.crop(
+        (x + x_offset, y + y_offset, x + width + x_offset,
+         y + height + y_offset)) for x_offset, y_offset in offsets}
```
Comment (magdasta) [inline apps/rendering/resources/scripts/img_metrics_calculator.py]: Please refactor this method, it's hardly readable in this form. I think making it more procedural would help

**Comment by mplebanski** [comment]
Can we close this?

**Comment by weaselix** [comment]
Yes, we can.

**Pull request closed without merging**